nikola/plugins/basic_import.py

```diff
@@ -166,7 +166,7 @@ def write_metadata(filename, title, slug, post_date, description, tags, **kwargs
         with io.open(filename, "w+", encoding="utf8") as fd:
             data = {'title': title, 'slug': slug, 'date': post_date, 'tags': ','.join(tags), 'description': description}
             data.update(kwargs)
-            fd.write(utils.write_metadata(data))
+            fd.write(utils.write_metadata(data, metadata_format='nikola', comment_wrap=False))
```

Comment:
Every plugin has a `set_site` call which is called on plugin initialization. There is no documentation saying that this isn't the case for `Command` plugins, and in particular importer plugins. So I would assume it is a bug if the `site` object isn't there.